frontend/src/features/properties/upload/Instructions.tsx

```diff
@@ -0,0 +1,87 @@
+import './Instructions.scss';
+
+import React from 'react';
+import { Accordion } from 'react-bootstrap';
+
+const mandatoryHeaders =
+  'PID,Type (Land or Building),Classification,Name,Ministry,Location (City),Latitude,Longitude'.split(
+    ',',
+  );
+
+const optionalHeaders =
+  'Status,Description,Agency (subagency for Ministry),Address,Postal,Assessed Land Value,Land Assessment Year,Netbook Value,Assessed Building Value,Building Assessment Year,Land Area,Legal Description,Construction Type,Predominate Use,Tenancy,Rentable Area,Building Floor Count,Local ID'.split(
+    ',',
+  );
+
+/**
+ * @description A set of instructions for the CSV upload page
+ * @returns {React.FC} A div with a list of instructions.
+ */
+export const Instructions = () => (
+  <div id="instructions">
+    <h2>Upload Instructions</h2>
+    {/* STEPS */}
+    <h4>Steps</h4>
+    <ol>
+      <li>Drag and drop a file or select the file upload area to choose a file.</li>
+      <li>
+        Select the <b>Start Upload</b> button.
+      </li>
+      <li>Stay on the results page until the upload is complete.</li>
+      <li>
+        (Optional) Select <b>Download Results</b> to receive a CSV file with each PID and its
+        success status.
+      </li>
+    </ol>
+
+    {/* REQUIREMENTS */}
+    <h4>Requirements</h4>
+    <ul>
+      <li>
+        File must be <b>.csv</b> format.
+      </li>
+      <li>
+        CSV file should contain the following headers: Required Headers cannot have blank values.
+        <Accordion flush id="required-headers">
+          <Accordion.Item eventKey="0">
+            <Accordion.Header>Required Headers</Accordion.Header>
+            <Accordion.Body>
+              <ul>
+                {mandatoryHeaders.map((header) => (
+                  <li key={header}>{header}</li>
+                ))}
+              </ul>
+            </Accordion.Body>
+          </Accordion.Item>
+        </Accordion>
+        Optional Headers may have blank values.
+        <Accordion flush id="optional-headers">
+          <Accordion.Item eventKey="0">
+            <Accordion.Header>Optional Headers</Accordion.Header>
+            <Accordion.Body>
+              <ul>
+                {optionalHeaders.map((header) => (
+                  <li key={header}>{header}</li>
+                ))}
+              </ul>
+            </Accordion.Body>
+          </Accordion.Item>
+        </Accordion>
+      </li>
+    </ul>
+
+    {/* CAVEATS */}
+    <h4>Caveats</h4>
```

Comment:
```suggestion
    <h4>Notes</h4>
```
Lindsay asks for a wording change here + word colour red.

Reply:
Adding now